cmd/utils.go

```diff
@@ -28,21 +28,30 @@ func userCache(pc string) string {
 	return pkgrCacheDir
 }
 
-func getWorkerCount() int {
+func GetWorkerCount() int {
```

Comment:
whats the point of having these as exported functions? eg GetWorkerCount vs getWorkerCount. Likewise, why just return another function and not use GetRestrictedWorkerCount directly. If you're going to abstract, you want some more logic to be at play.

Reply:
The abstraction was to make unit-testing easier. I wanted to keep the "complexity" (however minimal) of calling runtime.numCPUs at this level of code rather than push it up a layer, as I think the upper layers are already a bit cluttered. I am still of this opinion, but do you want me to change it anyway?

As for exporting, that was me thinking this was Java and that I wouldn't be able to unit-test from another file unless it was exported. I'll change that, but I want to leave the abstraction.